### PR TITLE
[NETBEANS-1074] Module Review web.jsf20

### PR DESCRIPTION
**Pull request by sarveshkesharwani**
- Mavenization of jsf-2.2.zip to javax.faces-2.2.7.jar
- Added CDDL-1.1 license for javax.faces-2.2.7.jar

### CODE REVIEW

File: enterprise/web.jsf20/external/binaries-list [hunk 1]
```diff
@@ -1 +1 @@
-99277566601C4D5C2598B0206B5DC071932F3641 jsf-2.2.zip
+94AE75EC83EF51C2F94C17D7C0F85E7C6518CC71 org.glassfish:javax.faces:2.2.7
```
Comment (junichi11) [inline enterprise/web.jsf20/external/binaries-list]: Missing the license header?

Comment (sarveshkesharwani) [reply enterprise/web.jsf20/external/binaries-list]: @junichi11 , Thanks for pointing this out. It was an oversight on my part.

**Comment by sarveshkesharwani** [comment]
@matthiasblaesing , Thanks for the review. I have removed the file jsf20ri.xml, and made the other changes in the latest commit.

**Comment by matthiasblaesing** [comment]
@sarveshkesharwani thank you - merging.

**Pull request merged**